### ISSUE TITLE
Initial Model  -1 (Overfitting Problem)

### ISSUE THREAD

**Issue by hrshtsharma17** [hrshtsharma17/ToothdecayNN#1]
The model has low validation accuracy and high accuracy over training data. The confusion matrix plotted failed to give any useful results. Consider solutions for overfitting.
